src/zed_cpu.cpp

```diff
@@ -0,0 +1,140 @@
+#include <zed_cpu.hpp>
+
+#include <memory>
+#include <vector>
+
+#include <cv_bridge/cv_bridge.h>
+#include <image_transport/image_transport.hpp>
+#include <opencv2/opencv.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <sensor_msgs/msg/imu.hpp>
+
+#include <zed_lib/sensorcapture.hpp>
+#include <zed_lib/videocapture.hpp>
+
+namespace zed_cpu
+{
+
+ZedCameraNode::ZedCameraNode() : Node("zed_camera", "zed_camera")
+{
+  // ROS initialization
+  left_image_pub_ = image_transport::create_publisher(this, "rgb/left_image");
+  right_image_pub_ = image_transport::create_publisher(this, "rgb/right_image");
+
+  imu_pub_ = create_publisher<sensor_msgs::msg::Imu>("imu_data", 1);
+
+  CameraInit();
+  SensorInit();
+
+  RCLCPP_INFO(get_logger(), "Node started");
```

Comment:
```suggestion
  imu_pub_ = this->create_publisher<sensor_msgs::msg::Imu>("imu_data", 1);

  CameraInit();
  SensorInit();

  RCLCPP_INFO(this->get_logger(), "Node started");
```
It is generally advised to include "this"  when calling member functions of current instance. It improves code clarity, and helps avoiding ambiguity (eg. class having function with the same name). Sorry for confusion with `get_logger`.